app/helpers/container_service_helper/textual_summary.rb

```diff
@@ -9,19 +9,21 @@ def textual_group_properties
       creation_timestamp
       resource_version
       session_affinity
+      service_type
       portal_ip
     )
   end
 
   def textual_group_port_configs
-    labels = [_("Name"), _("Port"), _("Target Port"), _("Protocol")]
+    labels = [_("Name"), _("Port"), _("Target Port"), _("Protocol"), ("Node Port")]
```

Comment:
Why is `("Node Port")` without a leading underscore like the other items in this array?